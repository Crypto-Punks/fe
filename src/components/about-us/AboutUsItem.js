import React from 'react';
import PropTypes from 'prop-types';

const Person = ({ name, spiritAnimal, image, blurb, website }) => {
  return (
    <div>
      <h1>{name}</h1>
      <h2>Spirit Animal: {spiritAnimal}</h2>
      <img src={image} alt={name}/>
      <a target='_blank' rel="noopener noreferrer" href={website}>{`${name}'s Website`}</a>
      <p>{blurb}</p>
    </div>
  );
};

Person.propTypes = {
  name: PropTypes.string.isRequired,
  spiritAnimal: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  website: PropTypes.string.isRequired,
  blurb: PropTypes.string.isRequired
};

export default Person;
