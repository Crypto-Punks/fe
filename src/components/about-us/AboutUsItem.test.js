import React from 'react';
import { shallow } from 'enzyme';
import AboutUsItem from './AboutUsItem';

const data = {
  name: 'test',
  spiritAnimal: 'test',
  image: 'test',
  blurb: 'test'
};

describe('AboutUsItem component', () => {
  it('renders AboutUsItem', () => {
    const wrapper = shallow(<AboutUsItem { ...data} />);
    expect(wrapper).toMatchSnapshot();
  });
});
