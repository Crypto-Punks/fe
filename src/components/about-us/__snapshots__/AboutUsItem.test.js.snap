// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUsItem component renders AboutUsItem 1`] = `
<div>
  <h1>
    test
  </h1>
  <h2>
    Spirit Animal: 
    test
  </h2>
  <img
    alt="test"
    src="test"
  />
  <p>
    test
  </p>
</div>
`;
